webpack/containers/Header.js

```diff
@@ -9,9 +9,18 @@ import NotificationDropdown from '../components/NotificationDropdown';
 import NotificationMenu from '../components/NotificationMenu';
 import { fetchNotifications } from '../actions/notification_actions';
 
-let LoginMenu = ({logInOpener, signUpOpener, currentUser}) => {
+let LoginMenu = ({openIdServer, logInOpener, signUpOpener, currentUser}) => {
   let loggedIn = ! _.isEmpty(currentUser);
   if(!loggedIn) {
+    if(openIdServer != '') {
+      return (
+        <ul className="nav navbar-nav">
+          <li className="log-in-link">
+            <a tabIndex="2" href={openIdServer}>Login</a>
```

Comment:
I don't know what OPIDC_HOST contains. However, it is used directly in an href attribute, so I assumed it was a URL. Has this code been tested with an actual OIDC setup?